test/verify/check-system-info

```diff
@@ -947,6 +947,9 @@ password=foobar
         b.wait_text("#crypto-policy-button", "FIPS")
         self.assertEqual(m.execute("cat /proc/sys/crypto/fips_enabled").strip(), "1")
 
+        m.execute(cmd + " --set FIPS:OSPP")
+        b.wait_not_present("#inconsistent_crypto_policy")
```

Comment:
This is pretty much a no-op. `wait_not_present` always needs to be preceeded by some "positive" change of that --set action. I suggest to reorder this after the `--set DEFAULT` below -- then do the `FIPS:OSPP` and wait for the UI to show "FIPS", and *then* the `self.assertFalse(b.is_present(#inconsistent))` makes sense.